scripts/feature_tests/README.md

```diff
@@ -117,13 +116,26 @@ export NUM_NODES=4
 while **Centos** uses:
 
 ```bash
-export CAPM3_VERSION=v1alpha5
 export IMAGE_OS=Centos
 export EPHEMERAL_CLUSTER=minikube
 export CONTAINER_RUNTIME=podman
 export NUM_NODES=4
 ```
 
+Both  **Ubuntu** and **Centos** setup for feature tests use:
+
+```bash
+export CAPM3_VERSION=v1alpha5
+export CAPI_VERSION=v1alpha4
+```
+
+while upgrade tests use:
+
+```bash
+export CAPM3_VERSION=v1alpha4
+export CAPI_VERSION=v1alpha3
```

Comment:
Maybe
```suggestion
export CAPM3_VERSION=v1alpha5
export CAPI_VERSION=v1alpha4
```
since you are updating it in https://gerrit.nordix.org/c/infra/cicd/+/9983/ ?

Reply:
I have to do modifications for that in `upgrade.yml` anyways and will update those with that change, since this is feature tests related only.